README.md

```diff
@@ -2,10 +2,43 @@
 
 The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
 
+The TSC is community and contributor driven and is a home for community managed 
+projects and assets. The mission of the Node.js Foundation is to enable and encourage 
+the adoption of Node.js and the TSC is the home of the community driven portion
+of executing that mission.
+
+Since the Foundation and the TSC are intended to support the community the size and scope
+of the work and projects in it will grow to accomodate the needs of the community.
+
```

Comment:
This statement assumes that growing the size of the size and scope of the work and projects is the only / best way to support the community. It also implies unconditional growth of the scope as a mission of the TSC. 

As we discussed previously, diluting the limited resources is one of the potential drawbacks of such unconditional growth. 

Not objecting to growth in principle but I think that the passage from the *what* (the mission of the TSC, fostering the Node.js ecosystem) to the *how* needs to be further elaborated and this statement may fail to capture the complexity of the criteria involved.

I would either leave it out, or soften it a bit, for example to something like  "it *may* grow to accommodate the needs of the community, within the constraints of available resources and in accordance to the priorities determined by the TSC"